core/src/avm2/class.rs

```diff
@@ -370,6 +372,7 @@ impl<'gc> Class<'gc> {
                     // A 'callable' class doesn't have a signature - let the
                     // method do any needed coercions
                     vec![],
+                    Multiname::any(activation.context.gc_context),
```

Comment:
not PR related: yikes, we really should make `Any` not allocate at all, maybe replacing it by an Option-like enum. (IIRC avmplus uses a nullptr to represent an Any namespace and Any name)